<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/header-attrs-2.16/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/yeti.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-118976541-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-118976541-1');
</script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Peter Euclide</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Research.html">Research</a>
</li>
<li>
  <a href="ed_and_out.html">Education and Outreach</a>
</li>
<li>
  <a href="CV.html">CV</a>
</li>
<li>
  <a href="sample_map.html">Where I work</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<p><img src="photos/purdueBoat.jpg" style="width:100%;" align = "top"/></p>
<div
style="display: grid; grid-template-columns: 60% 40%; grid-column-gap: 10px; ">
<div>
<hr />
<div id="molecular-drivers-of-ecotypic-divergence"
class="section level1">
<h1>Molecular drivers of ecotypic divergence</h1>
<hr />
<p>Populations are often comprised of discrete ecotypes with distinct
morphology or behavior in the absence of reproductive isolation. One of
my research objectives is to identify both the molecular mechanisms and
environmental drivers of ecotypic divergence. Understanding how local
adaptation within a population can lead to diverged ecotypes has
important implications for speciation research and the conservation
diversity. My work uses molecular techniques, such as RAD-sequencing and
whole-genome-resequencing, to identify putatively adaptive variance
among ecotypes or populations. Using these approaches, I currently
involved in research that is identifying SNPs that are linked to lipid
content in F2 experimental lake trout crosses (left) and helping to
uncover how structural variants contribute to spawning ecotypes in
sockeye salmon. In my own lab, I hope to use these same techniques
research river-resident and river-transient smallmouth bass ecotypes
which have been repeatedly observed throughout the Great Lakes.</p>
</div>
</div>
<div>
<p><br/><br/><br/><br/></p>
<div class="figure">
<img src="photos/fat_cors.png" alt="" />
<p class="caption">Correlation between the number of positive effect
alleles at loci putatively associated with lipid content in lake trout
ecotypes.</p>
</div>
</div>
</div>
<hr />
<div
style="display: grid; grid-template-columns: 40% 60%; grid-column-gap: 10px ">
<div>
<p><br/></p>
<p><img src="photos/walleye.jpg" /></p>
<p><img src="photos/IMG_4655.JPG" /></p>
</div>
<div>
<hr />
<div id="improving-management-decisions-with-genetic-data"
class="section level1">
<h1>Improving management decisions with genetic data</h1>
<hr />
<p>Recreational and commercial fisheries are a valuable industry
throughout the world. Exploited fish populations are exposed to a suite
of anthropogenic influences often not experienced by other species, such
as direct harvest, stocking, and translocations. In the Great Lakes,
walleye support a large fishery which has resulted in a patchwork of
natural, stocked, and recovering spawning populations or stocks, each
contributing differently to the overall population abundance. The
complex combination of naturally reproducing and recovering stocks
require managers to consider multiple discrete spawning populations when
estimating stock recruitment and determining sustainable harvest quotas.
My research aims to (1) understand how discrete walleye stocks
throughout the Great Lakes contribute to a large Great Lakes wide
metapopulation, and (2) how spawning site fidelity and dispersal help to
sustain exploited populations and result in local adaptation.</p>
</div>
</div>
</div>
<hr />
<div
style="display: grid; grid-template-columns: 60% 40%; grid-column-gap: 10px; ">
<div>
<hr />
<div id="genetic-consequences-of-habitat-fragmentation"
class="section level1">
<h1>Genetic consequences of habitat fragmentation</h1>
<hr />
<p>Humans have altered the natural connectivity of fish populations
through constructing damns and causeways that limit fish dispersal and
ultimately gene flow which can lead to population sub-structuring across
a species range. Many barriers affecting species throughout North
America were built recently, and so I work at intercept between
behavioral ecology and evolutionary biology using molecular techniques,
long-term demography, and mark-recapture datasets to understand how
populations have adjusted to lost connectivity. In Lake Champlain, I
used population genetic and fisheries data to describe patterns of
neutral genetic structure and population demographics of fish native to
Lake Champlain which has been fragmented by seven causeways built during
the early twentieth century.</p>
</div>
</div>
<div>
<p><img src="photos/DSC_0447.jpg" /></p>
</div>
</div>
<hr />
<div
style="display: grid; grid-template-columns: 40% 60%; grid-column-gap: 10px; ">
<div>
<p><br/><br/></p>
<p><img src="photos/mysis2.jpg" /></p>
</div>
<div>
<hr />
<div id="partial-diel-vertical-migration" class="section level1">
<h1>Partial diel vertical migration</h1>
<hr />
<p>My interest in behavioral ecology stems from my thesis research on
partial diel vertical migration behavior of <em>Mysis diluviana</em> in
Lake Champlain. Using a combination of stable isotopes and basic
demographic data I determined that gravid females and large
<em>Mysis</em> with higher C:N content (lipid content) were more likely
to remain on the bottom at night while the rest of the population
migrated to the surface to feed.</p>
</div>
</div>
</div>
<hr />
<div
style="display: grid; grid-template-columns: 60% 40%; grid-column-gap: 10px; ">
<div>
<hr />
<div id="additional-research" class="section level1">
<h1>Additional research</h1>
<hr />
<p>Harmful cyanobacteria blooms are a global issue. Because
cyanobacteria have little nutritional value for zooplankton, it is
possible that higher trophic levels such as fish are impacted by the
influx of nutrition poor cyanobacteria at the base of the food web.
Working with a colleague trained in lipid analysis, we conducted a
two-month experiment to determine how essential fatty acid poor diets,
like what might happen during a long cyanobacteria bloom, impact fish
fatty acid content, swimming performance and respiration.</p>
</div>
</div>
<div>
<iframe src="https://player.vimeo.com/video/170673352" width="300" height="300" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
<p>
<a href="https://vimeo.com/170673352">JunkFoodHypothesis</a> from
<a href="https://vimeo.com/user38109713">Victoria Pinheiro</a> on
<a href="https://vimeo.com">Vimeo</a>.
</p>
</div>
</div>
<hr />

<p>2021 Peter Euclide, peuclide@purdue.edu. All rights reserved.</p>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
